lib/linguist/languages.yml

```
@@ -4381,6 +4382,64 @@ Nunjucks:
   tm_scope: text.html.nunjucks
   ace_mode: nunjucks
   language_id: 461856962
+OASv2:
+  type: programming
```

These are not programming languages, they're specifications written in JSON or YAML, which are both data types, so this should be `data` too. If a repo owner wants the name to appear in the language stats, they can implement an override.

```suggestion
  type: data
```